lib/actionButtons.test.js

```diff
@@ -104,78 +104,77 @@ describe('The "Task Complete" button should', () => {
 
 describe('The "Prior Task" button should', () => {
     const details = { terminus: false };
-    describe('If the task is set to manual', () => {
-        const manual = true;
-        test('and if !isOldestTask in the breadcrumbs, should return an enabled button with the getPreviousTask onclick action', () => {
-            const taskBreadcrumbs = ['task0', 'task1', 'task2'];
-            const workflowState = { currentTask: { id, manual, details }, taskBreadcrumbs };
-            expect(priorTaskButton(workflowState))
-                .toMatchObject({
-                    enabled: true,
-                    onClickAction: actions.getPreviousTask
-                });
-        });
-        test('and if isOldestTask in the breadcrumbs, should return a disabled button', () => {
-            const taskBreadcrumbs = ['task1', 'task2'];
-            const workflowState = { currentTask: { id, manual, details }, taskBreadcrumbs };
-            expect(priorTaskButton(workflowState))
-                .toMatchObject({ enabled: false, onClickAction: null });
-        });
-    });
-    describe('If the task is not set to manual', () => {
-        const manual = false;
-        test('and if breadcrumbs length > 1, should return an enabled button with the getPreviousTask onclick action', () => {
+    const manual = true; // TODO: confirm we no longer care if a task is manual or not
```

Comment:
@jonathansberry can you confirm this please

Reply:
For the prior task button, no we do not care if it is manual.